src/main/scala/tutorial/HelloCollision.scala

```diff
@@ -124,7 +124,7 @@ class HelloCollision extends SimpleApplication with ActionListener {
       case Right => characterDirection = characterDirection.copy(right = value)
       case Up => characterDirection = characterDirection.copy(up = value)
       case Down => characterDirection = characterDirection.copy(down = value)
-      case Jump => player.jump()
+      case Jump =>  if (value) { player.jump(new Vector3f(0,20f,0));}
```

Comment:
Thanks, I'll try to run it tonight. Is the `;` necessary?

Reply:
I wouldn't mind seeing such semi-colons go but in this case the `{` `}` are good - otherwise hard to tell where the if-branch ends, so it is a code smell even if it isn't required (and it may be - I hope so).

Reply:
looks like Scala probably just runs the first statement after the `if`; I guess that is fine but I feel like it can sometimes look less than obvious when reading code.